libs/eo/emissions/shell/src/lib/eo-emissions-shell.module.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  EoEmissionsPageShellComponent,
+  EoEmissionsPageShellScam,
+} from './eo-emissions-page-shell.component';
+import { RouterModule, Routes } from '@angular/router';
+import { NgModule } from '@angular/core';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: EoEmissionsPageShellComponent,
+  },
```

Comment:
```suggestion
  { path: '', component: EoEmissionsPageShellComponent },
```

What do you think about this formatting instead of the 4-line one? :)

Reply:
Regarding the logo or "banner" for the ministry. There is now 2 different versions of it. One for this page, and one for Origin of Energy. Should we maybe press UX about this so the design uses the same image perhaps? :) What do you think?

Reply:
I have no opinion on this - Feel free to add this in one line, that is fine by me :-)